### PR TITLE
Fix error on invoicing period closed

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
same issue here, thanks :)

File: wrappers/odoo_client.py [hunk 1]
```diff
@@ -35,8 +36,19 @@ def drop_attendances(self, date_window):
              ('check_out', '=', None)
              ]
         )
+        error_removing_all = False
```
Comment (simahawk) [inline wrappers/odoo_client.py:39]: can you add a TODO? We should avoid dropping attendances if they don't change.
We could read them all 1st for the given week and delete only the ones not matching.

**Comment by simahawk** [comment]
tested: it works :+1:

File: wrappers/odoo_client.py (hunk 1 above)
Comment (TDu) [reply wrappers/odoo_client.py:39]: Done

Comment (simahawk) [reply wrappers/odoo_client.py:39]: where? :D

**Comment by simahawk** [comment]
merged manually, comment added

**Pull request closed without merging**